controllers/reservations.js

```diff
@@ -1,13 +1,21 @@
 var express = require('express');
+var router = express.Router();
 var Reservation = require('../models/reservation');
 
-var router = express.Router();
+// Get a reservation by id
+router.get("/:id", function(req, res) {
+  Reservation.get(req.params.id).then(function(reservation) {
+    res.status(200).json(reservation);
+  }).catch(function(error) {
+    res.status(500).json({error: error});
+  });
+});
 
 // Create and reserve a new reservation
 router.post('/', function(req, res) {
   Reservation.reserve(req.body.userId, req.body.latitude, req.body.longitude)
     .then(function(reservation) {
-      res.json(reservation);
+      res.status(201).json(reservation);
```

Comment:
On each of these functions we return the reservation object in JSON format, as opposed to the other files where we return the JSON of {spot: spot}. Is there a functional difference?